react/spring-page/src/App.js

```diff
@@ -0,0 +1,27 @@
+import './App.css';
+import React from 'react';
+import {
+  Route, BrowserRouter, Switch, Redirect,
+} from 'react-router-dom';
+import { Header } from './components/Header/Header';
+import { Main } from './components/Main/Main';
+import { LoginForm } from './components/LoginForm/LoginForm';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Redirect to="/login" />
+      <div className="App">
+        <Switch>
+          <Route path="/main" component={Header}>
+            <Header />
+            <Main />
+          </Route>
```

Comment:
You are putting different components in route content. Leave one of them.

Reply:
got it